### PR TITLE
Add empty volume to some templates and remove some unused files

### PR DESCRIPTION
**Pull request by xiuwang**

### CODE REVIEW

File: build/perl516-imageupgrade-stibuild.json [hunk 1]
```diff
@@ -142,6 +147,9 @@
             "name": "origin-perl516-sample:latest"
           }
         },
+        "postCommit": {
+          "args": ["bundle", "exec", "rake", "test"]
```
Comment (wzheng1) [inline build/perl516-imageupgrade-stibuild.json]: I don't think this json file works well if this postCommit line exists.

**Comment by wzheng1** [comment]
Seems you didn't check whether the update templates works, since if there is line "args": ["bundle", "exec", "rake", "test"] in "postCommit", build maybe failed for images other than ruby; and there are some grammar issue too: 
]$ oc new-app -f https://raw.githubusercontent.com/xiuwang/v3-testfiles/master/build/perl516rhel7-env-sti.json
error: unable to load template file "https://raw.githubusercontent.com/xiuwang/v3-testfiles/master/build/perl516rhel7-env-sti.json": unable to decode "https://raw.githubusercontent.com/xiuwang/v3-testfiles/master/build/perl516rhel7-env-sti.json": couldn't get version/kind; json parse error: invalid character '{' after object key:value pair

**Comment by xiuwang** [comment]
comment addressed.

File: build/ruby20rhel7-template-docker.json [hunk 2]
```diff
@@ -128,7 +128,7 @@
             "from": {
               "kind": "ImageStreamTag",
               "name": "ruby:2.0",
-              "namespace": "openshift"
+              "namespace": "openshift",
```
Comment (wzheng1) [inline build/ruby20rhel7-template-docker.json]: "," is unnecessary here

**Pull request merged**